ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.10.yaml

```diff
@@ -53,7 +53,7 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-azure-ipi-proxy-techpreview
+    workflow: cucushift-installer-rehearse-azure-ipi-private
```

Comment:
I do not think 4.10 private install would pass before the fix of 2082395 is backported. But from the job result, it get passed, obviously there is something wrong in the logic.

And the more important thing is even the code logic get fixed in ipi-conf-azure, but it will affect <=4.10 private cluster install.

Reply:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/29967/rehearse-29967-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.10-installer-rehearse-azure/1547532843159130112/artifacts/ci-operator.log
error msg=  54: variable \"azure_base_domain_resource_group_name\" {\nlevel=error\nlevel=error msg=The root module input variable \"azure_base_domain_resource_group_name\" is not\nlevel=error msg=set, and has no default value. Use a -var or -var-file command line argument\nlevel=error msg=to provide a value for this variable.